spec/auth_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe Strava do
+  describe 'auth' do
+    it 'shuts down server when launchy fails' do
+      options = { client_id: 'foo', client_secret: 'bar' }
```

Comment:
since `options` is not reused more than once, just write them into the arguments of `Strava::Cli::Auth.new`